Podfile

```diff
@@ -175,7 +175,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.84.0'
+  gutenberg commit: '16130b31389066fbbb06afa116b5b3ab5591fa38'
```

Comment:
Friendly note: we'll need to update this to the newly created `1.84.1` tag before merging.